autora/experimentalist/sampler.py

```diff
@@ -0,0 +1,52 @@
+import random
+from typing import Iterable, Sequence, Union
+
+import numpy as np
+from alipy.query_strategy.query_labels import QueryInstanceUncertainty
```

Comment:
Hey @gtdang  – alipy seems broken when I run it, and I don't think we necessarily want to have it as a dependency. How easy would it be for us to do our own uncertainty sampler? It looks pretty straightforward.

Reply:
Looks like it was a python 3.9+ issue – ALiPy doesn't work with 3.9 or 3.10. Same thing – I think we need to think about whether we want to keep it as a dependency.